service_mesh/v2x/prepare-to-deploy-applications-ossm.adoc

```diff
@@ -14,6 +14,13 @@ ifdef::openshift-enterprise[]
 For more information, see the {product-title} documentation, xref:../../applications/deployments/what-deployments-are.adoc[Understanding deployments].
 endif::[]
 
+[NOTE]
+====
+Traffic started by Init Containers, specialized containers that run before the application containers in a pod, cannot travel outside of the service mesh by default. Any action Init Containers perform that requires establishing a network traffic connection outside of the mesh fails.
+
+For more information about connecting Init Containers to a service, see the Red Hat Knowledgebase solution link:https://access.redhat.com/solutions/6963263[initContainer in CrashLoopBackOff when using Service Mesh and sending traffic to a host which requires DNS resolution].
```

Comment:
@rh-tokeefe We are supposed to link only to Verified knowledgebase artices.

Reply:
@rh-tokeefe I don't know the process (perhaps your support team connection can explain), but as I understand these articles get verified:

![image](https://github.com/openshift/openshift-docs/assets/25209155/31e920b9-2f48-48a4-b298-e8980e6f4abc)

![image](https://github.com/openshift/openshift-docs/assets/25209155/df9e7330-3043-47a9-8937-b5e1c3317500)

We're not supposed to link to the Unverified articles, similar to how we don't publish any docs without QE review and approval.

Reply:
Contacted the Supported Engineer who helped with the verified solution. PR has been revised.